recipes/fruit/all/conanfile.py

```diff
@@ -12,27 +18,15 @@ class FruitConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/google/fruit"
     license = "Apache-2.0"
-    topics = ("conan", "fruit", "injection")
+    topics = ("injection", "framework")
     settings = "os", "compiler", "build_type", "arch"
     options = {"shared": [True, False],
                "use_boost": [True, False],
                "fPIC": [True, False]}
     default_options = {"shared": False, "use_boost": True, "fPIC": True}
```

Comment:
using `with_boost` sounds more consistent with the rest of the recipes than `use_boost`

Reply:
👍  on this good catch!

Reply:
True, but it had always been like this.
And I guess fruit in their CMake call it like this.
So do we change it? Would break how people are used to use this package.

Reply:
We can [deprecate the option](https://github.com/conan-io/conan-center-index/blob/c1361ec2150a7dcb966608c39b6c12bc34e07858/docs/faqs.md?plain=1#L166) and read it incase it's used  to help with migration

No this is not required but ideally

Reply:
I would propose to close this PR with this change the way it is and the  when I add fruit 3.7.1, I will adapt accordingly. Agreed?